src-java/kilda-persistence-hibernate/src/main/java/org/openkilda/persistence/hibernate/entities/history/HibernateFlowEvent.java

```diff
@@ -74,8 +74,14 @@ public class HibernateFlowEvent extends EntityBase implements FlowEventData {
     @Column(name = "unstructured", columnDefinition = "json")
     private FlowEventUnstructured unstructured;
 
+    /**
+     * The ordering relies on grouping messages in topologies: history messages for the same correlation ID arrive to
+     * the same worker and processed sequentially within a single operation (see fieldsGrouping in topology builders).
+     * However, timestamps are not generated by DB, so assumptions about the relation between timestamp and
+     * id cannot be made.
+     */
     @OneToMany(mappedBy = "event", cascade = CascadeType.ALL)
-    @OrderBy("event_time")
+    @OrderBy("eventTime , id")
```

Comment:
while it is possible to use both the column name or the field name as a value, it seems like we use snake case preferably, it might be better to stick to one style.

Reply:
I think its better to stick with the field name then. I chose this one because IDE can highlight typos, with the snake case it has false negatives, it underlines all fields as incorrect.